plugins/package-managers/gradle-inspector/src/main/kotlin/GradleInspector.kt

```diff
@@ -87,6 +87,11 @@ private val GRADLE_USER_HOME = Os.env["GRADLE_USER_HOME"]?.let { File(it) } ?: O
  */
 const val OPTION_GRADLE_VERSION = "gradleVersion"
 
+/**
```

Comment:
Commit message nits:
- Should we say "Maven" instead of "Gradle" artifacts?
- Typo in "rot" -> "root.